namenode/main.go

```diff
@@ -202,6 +200,9 @@ func (server *NameNode) ReportBlock(blockReport common.BlockReport, success *boo
 			slog.Debug("Create new storage info",
 				"file name", blockMetadata.FileName,
 				"block index", blockMetadata.BlockIndex)
+			for !(blockMetadata.BlockIndex < uint(len(inode.storageInfo))) {
+				inode.storageInfo = append(inode.storageInfo, blockStorageInfo{})
```

Comment:
Could you confirm that the LatestVersion field is set to 0 (invalid) by default here? Or I think explicitly write that out here is more safe.